src/redturtle/prenotazioni/interfaces.py

```diff
@@ -17,9 +18,33 @@ class ISerializeToPrenotazioneSearchableItem(Interface):
     """Prenotazione searchable item serializer interface"""
 
 
+class ISerializeToRetroCompatibleJson(Interface):
+    """Interface used to cereate the TEMPORARY retrocomattible serializers"""
+
+
+class IBookingReminderEvent(IObjectEvent):
+    """Booking reminder time arrived event"""
+
+
 class IPrenotazioneEmailMessage(Interface):
     """Prenotazione email message"""
 
 
-class ISerializeToRetroCompatibleJson(Interface):
-    """Interface used to cereate the TEMPORARY retrocomattible serializers"""
+class IBookingNotificationSender(Interface):
+    """Booking notification sender"""
+
+
+class IPrenotazioneSMSMessage(Interface):
```

Comment:
mixed Prenotazione/Booking in name interfaces.
I would standardize it.

Please add some extra infos in docstrings

Reply:
Done